File: pm_core/src/core/domain/entity/priority_entity.go
/*
Author: QuanTuanHuy
Description: Part of Serp Project
*/

package entity

type PriorityEntity struct {
	BaseEntity
	TenantID int64 `json:"tenantId"`

	Name        string  `json:"name"`
	Description *string `json:"description,omitempty"`
	IconUrl     *string `json:"iconUrl,omitempty"`
	Color       *string `json:"color,omitempty"`
	Sequence    int     `json:"sequence"`
	IsSystem    bool    `json:"isSystem"`
}

File: pm_core/src/main/java/serp/project/pmcore/core/domain/entity/PriorityEntity.java
/**
 * Author: QuanTuanHuy
 * Description: Part of Serp Project
 */

package serp.project.pmcore.core.domain.entity;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.experimental.SuperBuilder;

@Getter
@Setter
@SuperBuilder
@NoArgsConstructor
@AllArgsConstructor
public class PriorityEntity extends BaseEntity {
    private Long tenantId;
    private String name;
    private String description;
    private String iconUrl;
    private String color;
    private Integer sequence;
    private boolean isSystem;
}
